### ISSUE TITLE
Unable to update gem to 3.2.0

### ISSUE THREAD

**Issue by mjerem34** [stefankroes/ancestry#510]
Hello, it is impossible for me to update the version of the gem
I have this error on CircleCi and Heroku : 
```
rake aborted!
LoadError: cannot load such file -- /home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/ancestry-3.2.0/lib/ancestry/materialized_path_pg
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:325:in `block in require'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:291:in `load_dependency'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/activesupport-6.0.2.2/lib/active_support/dependencies.rb:325:in `require'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:49:in `require_relative'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/ancestry-3.2.0/lib/ancestry.rb:7:in `<main>'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/home/circleci/repo/config/application.rb:20:in `<top (required)>'
/home/circleci/repo/Rakefile:6:in `require_relative'
/home/circleci/repo/Rakefile:6:in `<top (required)>'
/home/circleci/repo/vendor/bundle/ruby/2.7.0/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
/usr/local/bin/bundle:23:in `load'
/usr/local/bin/bundle:23:in `<main>'
(See full trace by running task with --trace)```

**Comment by HectorMF** [comment]
I get the same error when running migrations after updating this gem

**Comment by jannemecek** [comment]
Same here, builds on Heroku failing today

**Comment by rvalladares77** [comment]
same here

**Comment by ezr-ondrej** [comment]
Can we yank the bad version?

**Comment by pollinoco** [comment]
Same error
rails aborted!
LoadError: cannot load such file -- /.rvm/gems/ruby-2.7.1/gems/ancestry-3.2.0/lib/ancestry/materialized_path_pg

**Comment by techpeace** [comment]
Just submitted #511 that drops the bad require. Though I am admittedly a novice to this lib, it at least avoids the loading error. If you'd like to use that version for the moment, you can use this in your `Gemfile`:

```ruby
gem "ancestry", github: "techpeace/ancestry", branch: "patch-1"
```

**Comment by kbrock** [comment]
thank you all for your help.
this has been released as `3.2.1`
